source/server/admin/stats_handler.cc

```diff
@@ -111,16 +111,21 @@ Http::Code StatsHandler::handlerStats(absl::string_view url,
     }
   }
 
+  Stats::Store& stats = server_.stats();
+  std::vector<Stats::ParentHistogramSharedPtr> histograms = stats.histograms();
+  stats.symbolTable().sortByStatNames<Stats::ParentHistogramSharedPtr>(
+      histograms.begin(), histograms.end(),
+      [](const Stats::ParentHistogramSharedPtr& a) -> Stats::StatName { return a->statName(); });
```

Comment:
I take it we'll end up doing a sort when we do chunking as well? And so we don't need an intercept test-only layer for this?

Reply:
The original point was we could have a middle-man that tests would invoke to get these values that would delegate to this, but sort it in output for the tests to consume -- and we wouldn't need to take the perf hit of sorting these in prod until we do the broader chunking change. I'm not sure how large this hit is, so maybe it's overkill.

But a test here that checks the output is sorted sgtm since that's now part of the contract.

Reply:
done